### PR TITLE
Add frontend handler for ecosystem vuln counts.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

File: gcp/appengine/frontend_handlers.py [hunk 1]
```diff
@@ -194,6 +191,17 @@ def osv_get_ecosystems():
   return sorted([bug.ecosystem[0] for bug in query if bug.ecosystem])
 
 
+@cache.cached(timeout=24 * 60 * 60)
```
Comment (rzhw) [inline gcp/appengine/frontend_handlers.py:194]: Do we need to set `key_prefix` here? https://flask-caching.readthedocs.io/en/latest/#caching-other-functions

**Review by rzhw — CHANGES_REQUESTED** [review]

File: gcp/appengine/main.py [hunk 2]
```diff
@@ -14,14 +14,15 @@
 """App Engine entrypoint."""
 
 import logging
-import os
 
 from flask import Flask
 import google.cloud.logging
 from google.cloud import ndb
 
+from cache import cache
```
Comment (jonathanmetzman) [inline gcp/appengine/main.py:22]: I think this is importing a variable from a module which I beleive is against the styleguide:
https://google.github.io/styleguide/pyguide.html#22-imports

File: gcp/appengine/cache.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Caching."""
+
+import os
+
+from flask_caching import Cache
```
Comment (jonathanmetzman) [inline gcp/appengine/cache.py:18]: This imports a class which I think is against the styleguide:
https://google.github.io/styleguide/pyguide.html#22-imports

**Review by jonathanmetzman — APPROVED** [review]
lgtm with nits

File: gcp/appengine/frontend_handlers.py (hunk 1 above)
Comment (rzhw) [reply gcp/appengine/frontend_handlers.py]: Thanks!

**Review by rzhw — APPROVED** [review]

File: gcp/appengine/main.py (hunk 2 above)
Comment (oliverchang) [reply gcp/appengine/main.py:22]: Great point, done!

File: gcp/appengine/cache.py (hunk 3 above)
Comment (oliverchang) [reply gcp/appengine/cache.py]: Great point, done!

**Pull request merged**